feature/session/src/main/res/layout/fragment_session_detail.xml

```diff
@@ -59,457 +59,477 @@
             android:visibility="invisible"
             />
 
-        <androidx.core.widget.NestedScrollView
+        <LinearLayout
+            android:id="@+id/content_parent"
+            android:orientation="vertical"
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
+            android:layout_height="wrap_content"
```

Comment:
I guess match_parent is good because this layout should fit the area anyway. 👀

Reply:
I agreed. I fixed it.